shell/platform/darwin/common/framework/Headers/FlutterMacros.h

```diff
@@ -15,6 +15,17 @@
 
 #endif  // defined(FLUTTER_SDK)
 
+#if defined(FLUTTER_FRAMEWORK) && \
+    FLUTTER_RUNTIME_MODE == FLUTTER_RUNTIME_MODE_DEBUG
+
+#define FLUTTER_DEBUG_EXPORT __attribute__((visibility("default")))
```

Comment:
I think this should probably be `#define FLUTTER_DEBUG_EXPORT FLUTTER_EXPORT`

Reply:
Updated.